sdk/iot/provisioning/inc/az_iot_provisioning_client.h

```diff
@@ -69,14 +71,14 @@ AZ_NODISCARD az_result az_iot_provisioning_client_init(
     az_span global_device_endpoint,
     az_span id_scope,
     az_span registration_id,
-    az_iot_provisioning_client_options* options);
+    az_iot_provisioning_client_options const* options);
 
 /**
- * @brief Gets the MQTT client id.
+ * @brief Gets the MQTT user name.

```

Comment:
nit: here it's two words later it's one. Unify one way or the other?

Reply:
Good point. I think we want username (single word).